modular_citadel/code/game/objects/items/defib.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/defibrillator
+	var/healdisk = FALSE // Will we shock people dragging the body?
+	var/pullshocksafely = FALSE //Dose the unit have the healdisk upgrade?
+	var/fastasfuck = 0 // is the defib faster
+	var/snowflake = 10
+
+/obj/item/twohanded/shockpaddles/shock_touching(dmg, mob/H)
+	if(defib.pullshocksafely && isliving(H.pulledby))
+		H.visible_message("<span class='danger'>The defibrillator safely discharges the excessive charge into the floor!</span>")
+		return
+	..()
+
+/obj/item/defibrillator/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/disk/medical/defib_heal))
+		if(healdisk)
+			to_chat(user, "<span class='notice'>This unit is already upgraded with this disk!</span>")
+			return TRUE
+		to_chat(user, "<span class='notice'>You upgrade the unit with Heal upgrade disk!</span>")
```

Comment:
I can't even into the sense of this

Reply:
It's to let people know that its upgraded. So they don't have to double click it to see if it is upgraded or not